FSharp.Compiler.Service.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio 15
```

Comment:
i think is good to add another `.sln` file.

```
dotnet new sln -n src/Fharp.compiler.netcore.sln
dotnet sln src/Fharp.compiler.netcore.sln add my.fsproj
```

because like that we can do `dotnet restore src/Fharp.compiler.netcore.sln` so that will restore all project, good for build script